interna/memberdb/migrations/0002_member_access_code.py

```diff
@@ -2,7 +2,12 @@
 
 
 from django.db import migrations, models
-import memberdb.access_codes
+try:
+    import memberdb.access_codes
+    generate = memberdb.access_codes.generate_access_code
+except ImportError:
+    def generate(*args, **kwargs):
+        return 'gone'
```

Comment:
Is this stuff automatically generated?

Reply:
No, I had to adjust the migration because the migrations would fail otherwise now that the `generate_access_code` function is gone... Seems to work though.